tgui/packages/tgui/interfaces/NtosSelfServe.tsx

```diff
@@ -0,0 +1,122 @@
+        <Stack>
+          <Stack.Item width="100%">
+            <SelfServePage />
+          </Stack.Item>
+        </Stack>
+      </NtosWindow.Content>
+    </NtosWindow>
+  );
+};
+
+const SelfServePage = (props) => {
+  const { act, data } = useBackend<Data>();
+  const {
+    authIDName,
+    authIDRank,
+    authCardHOPLocked,
+    authCardTimeLocked,
+    stationAlertLevel,
+    trimClockedOut,
+    trimAssignment,
+  } = data;
+
+  return (
+    <Section title="Enterprise Resource Planning">
+      <Section title={'Welcome ' + authIDName + '!'}>
+        <Stack wrap="wrap">
+          <Stack.Item width="100%" mt={1} ml={0}>
+            Current Assignment: {trimAssignment || '-----'}
+          </Stack.Item>
+          <Stack.Item align="center" mt={1} ml={0}>
+            Job Title: {authIDRank || '-----'}
+          </Stack.Item>
+          <Stack.Item width="100%" mt={1} ml={0}>
+            Current Status: {trimClockedOut ? 'Off-Duty' : 'Active Duty'}
+          </Stack.Item>
+          <Stack.Item width="100%" mt={1} ml={0}>
+            Station Alert Level: {capitalizeFirst(stationAlertLevel) || '-----'}
+          </Stack.Item>
+        </Stack>
+      </Section>
+
+      <Section title="Employee Self Serve">
+        <Stack wrap="wrap">
+          <Stack.Item width="100%" mt={1} ml={0}>
+            <Stack>
+              <Stack.Item>
+              {authCardHOPLocked ? (
+                <NoticeBox>
+                  This card is locked from changing duty status, please visit the
+                  HoP window to clock in/out.
+                </NoticeBox>
+              ) : (
+                <Button
+                  width="100%"
+                  disabled={authCardTimeLocked}
+                  onClick={() => act('PRG_change_status')}
+                >
+                  <center>
+                    {trimClockedOut
+                      ? 'Return to Job Assignment'
+                      : 'Clock Out from Job Assignment'}
+                  </center>
+                </Button>
+              )}
+              </Stack.Item>
+              <Stack.Item>
+                <Button
+                  right="0px"
+                  width="100%"
+                  icon='eject'
+                  onClick={() => act('PRG_eject_id')}
+                  />
+              </Stack.Item>
+            </Stack>
+          </Stack.Item>
+          {!trimClockedOut ? (
+            <Stack.Item width="100%" mt={1} ml={0}>
+              <NoticeBox info>
+                While off-duty any restricted items will be transferred to a
+                crew equipment lockbox, to be returned upon clocking in.
+              </NoticeBox>
+            </Stack.Item>
+          ) : (
+            ''
+          )}
+          <Stack.Item width="100%" mt={1} ml={0}>
+            <NoticeBox info>
+              Please note that security and command cannot punch in through this
+              application, and must go through an appropriate Command member.
```

Comment:
They can use the stationary console that has specific support for them

Reply:
When it was written the intention was that this program would replace both the time clock on the wall, and also be available on the PDA.